ports/atmel-samd/boards/ugame10/mpconfigboard.h

```diff
@@ -26,7 +26,7 @@
 #define SPI_FLASH_DIPO              3   // same as MISO pad
 
 // These are pins not to reset.
-#define MICROPY_PORT_A        (PORT_PA11 | PORT_PA16 |\
+#define MICROPY_PORT_A        (PORT_PA18 | PORT_PA16 |\
```

Comment:
`PORT_PA18` is in `MICROPY_PORT_A` twice.

Reply:
I didn't notice that, thanks. I wonder why PA18 was there already. Oh, well, fixed now.